api/router.go

```diff
@@ -66,16 +66,17 @@ func NewRouter(s *Server) *mux.Router {
 	r.Methods("POST").Path("/v1/volumes").Handler(f(schemas, s.fwd.Handler(NodeHasDefaultEngineImage(s.m), s.VolumeCreate)))
 
 	volumeActions := map[string]func(http.ResponseWriter, *http.Request) error{
-		"attach":             s.VolumeAttach,
-		"detach":             s.VolumeDetach,
-		"salvage":            s.VolumeSalvage,
-		"recurringUpdate":    s.VolumeRecurringUpdate,
-		"updateReplicaCount": s.VolumeUpdateReplicaCount,
-		"updateDataLocality": s.VolumeUpdateDataLocality,
-		"updateAccessMode":   s.VolumeUpdateAccessMode,
-		"activate":           s.VolumeActivate,
-		"expand":             s.VolumeExpand,
-		"cancelExpansion":    s.VolumeCancelExpansion,
+		"attach":                   s.VolumeAttach,
+		"detach":                   s.VolumeDetach,
+		"salvage":                  s.VolumeSalvage,
+		"recurringUpdate":          s.VolumeRecurringUpdate,
+		"updateReplicaCount":       s.VolumeUpdateReplicaCount,
+		"updateReplicaAutoBalance": s.VolumeUpdateReplicaAutoBalance,
+		"updateDataLocality":       s.VolumeUpdateDataLocality,
```

Comment:
nit: put a space around this, keep the replica stuff in their own sections and cleans up the diff.
I despise the go auto fmt for keeping them in line, just leads to messy diffs/git blame when trying to see when a field was added

Reply:
Thanks, addressed.